CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+### Added
+## 6.0.0 (Unreleased)
+-   `unitSpace` prop to `<ChannelValue>` to manage spacing between the value and units.
```

Comment:
Should write the changlog in Angular’s terminology …

Reply:
Updated readme file.